test/bye_bye_bye/utils_test.exs

```diff
@@ -263,4 +263,103 @@ defmodule ByeByeBye.UtilsTest do
       assert result == %{}
     end
   end
+
+  describe "protox_struct_to_map/1" do
+    test "converts a simple Protox struct to a map" do
+      struct = %FeedEntity{
+        id: "test_id",
+        is_deleted: false
+      }
+
+      result = Utils.protox_struct_to_map(struct)
+
+      assert is_map(result)
+      refute is_struct(result)
+
```

Comment:
nit: I feel like these assertions could be more succinctly expressed via pattern-matching